webui/web.go

```diff
@@ -231,7 +231,7 @@ func BasicAuth(pass http.HandlerFunc) http.HandlerFunc {
 			http.Error(w, "Authorization required", http.StatusUnauthorized)
 			return
 		}
-		if subtle.ConstantTimeCompare([]byte(password), []byte(Password)) == 1 {
+		if subtle.ConstantTimeCompare([]byte(password), []byte(Password)) == 0 {
```

Comment:
This should be != 1.  The function only returns 1 if the contents are equal.  It does not document what it returns when the contents don't match.